api/src/test/java/org/jmisb/api/klv/st0102/localset/SecurityMetadataLocalSetTest.java

```diff
@@ -146,7 +147,7 @@ public void testFullFromBuilder()
         byte[] bytes = fullMessage.frameMessage(false);
         // System.out.println(ArrayUtils.toHexString(bytes));
 
-        Assert.assertEquals(bytes[21], Classification.TOP_SECRET.getCode());
+        Assert.assertEquals(bytes[20], Classification.TOP_SECRET.getCode());
```

Comment:
This line changes because the length value is now only two bytes (instead of three previously), so the applicable value moves up.